css/styles.css

```diff
@@ -321,3 +402,290 @@ textarea {
   left: 4px;
   font-size: 11px;
 }
+
+.desktopnav {
+  visibility: hidden;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: lightgray;
+}
+
+.desktopnav ul li {
+  margin-right: 30px;
+}
+
+.awesomedeco1 {
+  background-image: url(../resources/image-geometry_7.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 1488px;
+  top: 296px;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+}
+
+.awesomedeco2 {
+  background-image: url(../resources/image-geometry_6.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+  left: -86px;
+}
+
+.imggeo2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imggeo5 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.fixGap1 {
+  margin-bottom: -5px;
+}
+
+.imgGroup83 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.redSquare {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imageDots2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+@media screen and (min-width: 1680px) {
```

Comment:
- [ ] The min-width for your desktop version is too big. Kindly edit to `768px`.

Reply:
Hi @ha-manel I've been working on your design requests all week. I've tried to pay close attention to the design and capturing a perfect rendition of the template. But I was wondering about this request. 

In the material handed to us by Microverse https://medium.com/level-up-web/best-practices-of-responsive-web-design-6da8578f65c4 it says that desktop begins at 1680px. Your size is for tablets. This affects the design. What do you say?